contrib/windows/julia-manifest.xml

```diff
@@ -7,6 +7,11 @@
       </requestedPrivileges>
     </security>
   </trustInfo>
+  <dependency>
+    <dependentAssembly>
+      <assemblyIdentity type='win32'  name='Microsoft.Windows.Common-Controls' version='6.0.0.0'     processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*' />
```

Comment:
I guess that I'm the one who is supposed to change the PR but can't find how to do it. It says that I "must be on a branch" to change the file but I can't see my branch and hence can't manage to change the file.